cmd/crates/soroban-spec-typescript/src/project_template/src/method-options.ts

```diff
@@ -2,21 +2,44 @@
 let responseTypes: 'simulated' | 'full' | undefined
 export type ResponseTypes = typeof responseTypes
 
+export type XDR = string
```

Comment:
How about XDR_BASE64?

Reply:
Sure, I like that more. Rebuilding snapshot, then I'll push it up.